### PR TITLE
New Version: ProaData.B2BClient version 1.1.0.1

### PR DESCRIPTION
**Pull request by sistemasproadata**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/71597)

### CODE REVIEW

File: manifests/p/ProaData/B2BClient/1.1.0.1/ProaData.B2BClient.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: ProaData.B2BClient
+PackageVersion: 1.1.0.1
```
Comment (Trenly) [inline manifests/p/ProaData/B2BClient/1.1.0.1/ProaData.B2BClient.installer.yaml:5]:
```suggestion
PackageVersion: 1.0.3.0
```

File: manifests/p/ProaData/B2BClient/1.1.0.1/ProaData.B2BClient.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: ProaData.B2BClient
+PackageVersion: 1.1.0.1
```
Comment (Trenly) [inline manifests/p/ProaData/B2BClient/1.1.0.1/ProaData.B2BClient.yaml:5]:
```suggestion
PackageVersion: 1.0.3.0
```

**Review by Trenly — CHANGES_REQUESTED** [review]
The version written to the Apps And Features table is 1.0.3.0. 

If this is a newer version than 1.1.0.0, leaving the PackageVersion as `1.1.0.1` will cause the application to not be detected by winget, and adding the `AppsAndFeaturesEntries` to the manifest will cause the version range mapping to work incorrectly, since `1.0.3.0 < 1.1.0.0`. If this is a newer version, you will need to remove version 1.1.0.0 as part of this PR and change the PackageVersion to 1.0.3.0

If this is an older version, you only need to change the PackageVersion to 1.0.3.0

**Review by sistemasproadata — COMMENTED** [review]
Reviewed

**Comment by sistemasproadata** [comment]
Close and reponen new one

**Pull request closed without merging**